ModComponent/Mapper/ItemMapper.cs

```diff
@@ -211,17 +229,13 @@ private static void PostProcess(ModBaseComponent modComponent)
 				{
 					eachMaterial.shader = meshRenderer.material.shader;
 					eachMaterial.shaderKeywords = meshRenderer.material.shaderKeywords;
-
-					if (eachMaterial.GetTexture("_dmg_texture") == null)
-					{
-						eachMaterial.SetTexture("_dmg_texture", eachMaterial.GetTexture("_MainTex"));
-					}
```

Comment:
This is important and cannot be removed.

Reply:
This is why I removed it.
```
Material 'rahatLokum (Instance)' with Shader 'Shader Forge/TLD_StandardSkinned' doesn't have a texture property '_dmg_texture'
Material 'roastedAlmonds (Instance)' with Shader 'Shader Forge/TLD_StandardSkinned' doesn't have a texture property '_dmg_texture'
Material 'solitudeCider (Instance)' with Shader 'Shader Forge/TLD_StandardSkinned' doesn't have a texture property '_dmg_texture'
Material 'strawberryPlombir (Instance)' with Shader 'Shader Forge/TLD_StandardSkinned' doesn't have a texture property '_dmg_texture'
Material 'tunaPate (Instance)' with Shader 'Shader Forge/TLD_StandardSkinned' doesn't have a texture property '_dmg_texture'
Material 'Standard (Instance)' with Shader 'Shader Forge/TLD_StandardSkinned' doesn't have a texture property '_dmg_texture'
Material 'Standard (Instance)' with Shader 'Shader Forge/TLD_StandardSkinned' doesn't have a texture property '_dmg_texture'```

Reply:
Well yes, food isn't supposed to have a damage property. This is for clothing.

Reply:
restored.